File: 3872-find-most-frequent-vowel-and-consonant/3872-find-most-frequent-vowel-and-consonant.cpp
class Solution {
public:
    int maxFreqSum(string s) {
        int maxs_v=0;
        int maxs_c=0;
        map<char,int>mp;
        for(char i : s){
            mp[i]++;
        }
        
       for (auto& i : mp) {
    if (i.first == 'a' || i.first == 'e' || i.first == 'i' || i.first == 'o' || i.first == 'u') {
        maxs_v = max(maxs_v, i.second);
    } else {
        maxs_c = max(maxs_c, i.second);
    }
}
        return maxs_v+maxs_c;
    }
};

File: 3872-find-most-frequent-vowel-and-consonant/3872-find-most-frequent-vowel-and-consonant.py
class Solution:
    def maxFreqSum(self, s: str) -> int:
        x=collections.Counter(s)
        max_v,max_c=0,0
        for char,freq in x.items():
            if char in 'aeiou':
                max_v=max(max_v,freq)
            else:
                max_c=max(max_c,freq)
        return max_v+max_c
        